tests/PKSim.Tests/Core/ProjectMapperSpecs.cs

```diff
@@ -151,7 +168,8 @@ public void should_return_a_project_with_the_expected_building_blocks()
          _newProject.All<Population>().ShouldContain(_population);
       }
 
-      [Observation]
+      // TODO ignoring this test until the classifications are implemented
```

Comment:
The observed data are now part of the classification hierarchy, so I'll un-ignore this test again when the map to model is implemented